<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Test Pass]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="34564" started-at="2018-08-07T07:57:03Z" finished-at="2018-08-07T07:57:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="34564" started-at="2018-08-07T07:57:03Z" finished-at="2018-08-07T07:57:37Z">
      <class name="student.MyMaven.APPLICATION">
        <test-method status="PASS" signature="launch()[pri:0, instance:student.MyMaven.APPLICATION@675d3402]" name="launch" is-config="true" duration-ms="31481" started-at="2018-08-07T13:27:03Z" finished-at="2018-08-07T13:27:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="FAIL" signature="testScript()[pri:0, instance:student.MyMaven.APPLICATION@675d3402]" name="testScript" duration-ms="80" started-at="2018-08-07T13:27:34Z" finished-at="2018-08-07T13:27:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.fail(Assert.java:100)
	at student.MyMaven.APPLICATION.testScript(APPLICATION.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test Pass]]>
            </line>
          </reporter-output>
        </test-method> <!-- testScript -->
        <test-method status="PASS" signature="CLOSE(org.testng.ITestResult)[pri:0, instance:student.MyMaven.APPLICATION@675d3402]" name="CLOSE" is-config="true" duration-ms="2933" started-at="2018-08-07T13:27:34Z" finished-at="2018-08-07T13:27:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testScript status=FAILURE method=APPLICATION.testScript()[pri:0, instance:student.MyMaven.APPLICATION@675d3402] output=Test Pass]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CLOSE -->
      </class> <!-- student.MyMaven.APPLICATION -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
